### DISCUSSION TITLE
Getting bandwidth information from log drain

### DISCUSSION THREAD

**Discussion by tasgon** [vercel/community#5610] [Feature Question]
### Question

Is it possible to get bandwidth information from log drains? From what I can tell, it's possible to get information about the memory usage & duration of a request, but I don't see an obvious way of getting info on the incoming/outgoing bandwidth of a request.

**Comment by leerob** [comment]
Hey, this is not currently possible. Can you share more about what you're trying to do?

**Comment by tasgon** [comment]
I'm currently trying to set up an external logging/monitoring solution since Vercel monitoring does not give us alerts (and is a bit pricey), and currently my system works for everything except bandwidth data.
